extensions/README.md

```diff
@@ -39,19 +39,21 @@ If your block depends on another block, place them all in extensions folder:
 1. Use the [Jetpack Docker environment](https://github.com/Automattic/jetpack/tree/master/docker#readme).
 1. Start a new branch.
 1. Add your new extension's source files to the extensions/blocks directory.
-And add your extensions's slug the appropriate array in `extensions/index.json`.
+And add your extensions's slug the beta array in `extensions/index.json`.
+By keeping your extension in beta array, it's safe to do small PRs and merge frequently.
```

Comment:
I've noticed block PRs tend to grow big before merged, so I'd like to emphasize this beta-aspect and small PRs for easier iteration in case it's not clear for folks.